CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+@valentinyanakiev
```

Comment:
I believe you need an operator to indicate ownership - in your case, a wildcard
https://github.com/alkem-io/whiteboard-collaboration-service/blob/develop/.github/CODEOWNERS#L3